app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/Database/Workout/WorkoutDao.java

```diff
@@ -3,26 +3,38 @@
 import android.arch.lifecycle.LiveData;
 import android.arch.persistence.room.Dao;
 import android.arch.persistence.room.Insert;
+import android.arch.persistence.room.OnConflictStrategy;
 import android.arch.persistence.room.Query;
+import android.arch.persistence.room.TypeConverter;
+import android.arch.persistence.room.TypeConverters;
 
+import java.util.Date;
 import java.util.List;
 
+import edu.drexel.lapcounter.lapcounter.backend.TimestampConverter;
 
 
 @Dao
 public interface WorkoutDao {
 
-    @Insert
-    void addWorkout(Workouts workout);
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
```

Comment:
Is there a reason for this conflict strategy? from the definition of `Workout`, the ID is an autoincrement field so you wouldn't be specifying the ID on insert anyway.

Reply:
EDIT: I see in a different file that you use this for testing. I guess this is okay for now.

Reply:
Until we can generate and use data completely within the app, I think its best to leave it just for having some known entities in the system to check.  However, once we get that, I'm fine removing it and just adding in a update call instead.